source/api_reference/version-history.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Version history
+weight: 0
+---
+# Version history
+
+## Version 1
+
+Version 1 can be accessed using either of these base URLs:
+
+- `https://{register-name}.register.gov.uk/v1/`
+- `https://{register-name}.register.gov.uk/`
+
+## Next version
```

Comment:
It might also be good to introduce a short preamble describing exactly what the 'Next' API is: i.e. it's a term used for a 'preview' of the upcoming API, prior to that API becoming stable.